packages/client/utils/constants.ts

```diff
@@ -40,9 +40,14 @@ export const meetingColumnArray = [DONE, ACTIVE, STUCK, FUTURE] as TaskStatusEnu
 /* Scoping Task Search Filter */
 export const taskScopingStatusFilters = [ACTIVE, STUCK, FUTURE] as TaskStatusEnum[]
 
-/* Drag-n-Drop Items */
-export const TASK = 'task'
```

Comment:
This `TASK` constant is used in `ClientGraphQLServer.ts`  and `LocalAtmosphere.ts` and thus is replaced by `SubscriptionChannel.TASK`. No instances were used in dnd related places.